columnflow/inference/__init__.py

```diff
@@ -251,6 +251,7 @@ def category_spec(
         config_data_datasets: Sequence[str] | None = None,
         data_from_processes: Sequence[str] | None = None,
         mc_stats: float | tuple | None = None,
+        fill_empty_bins: float = 1e-5,
```

Comment:
`fill_empty_bins` sounds like something active, i.e. a function. Maybe `fill_emtpy_bins_with` would be more descriptive

Reply:
True, I will rename that :+1: